### PR TITLE
WIP: Implemented the code for Ionic CSS basic carousel slider

### PR DESCRIPTION
**Pull request by priyal-hotwax**
Closes #27 
https://app.clickup.com/t/1xx08jh

### Short Description and Why It's Useful
Implemented Ionic Scroll Animation with all functionality working including left and right button for navigating to scroll left to right using CSS transform and scroll snapping properties. There are some improvements and good to have as a feature as:
- On hovering on the carousel cards it would add some CSS or zoom-in-out.
-  Need to implement enabling and disabling in right and left button in carousel.

### Screenshots of Visual Changes before/after (If There Are Any)
**Before:**
Screenshot from 2022-01-11 12-31-19

**After:**
Screenshot from 2022-01-11 11-55-59

### CODE REVIEW

**Comment by priyal-hotwax** [comment]
Screenshot from 2022-01-11 11-55-59

**Comment by priyal-hotwax** [comment]
Short Demo here till what we have did so far:
https://www.loom.com/share/93218943c64848b7b39b63308b12e05f?sharedAppSource=personal_library

File: src/views/Home.vue [hunk 1]
```diff
@@ -64,9 +64,16 @@
...
+        <div class="scroller-wrapper" ref="ionCard">
+          <div @click="prevItems($event)" class="button-nav-wrapper prev-button-wrapper">
+            <ion-icon :icon="chevronBackOutline" />
+          </div>
+          <div @click="nextItems($event)" class="button-nav-wrapper next-button-wrapper">
+            <ion-icon :icon="chevronForwardOutline" />
+          </div>
+          <div class="scroller-content" ref="scrollItem">
+            <div class="scroller-item" v-for="i = 1 in 10" :key="i" >
+              <ion-card id="ion-card" class="ion-custom-card">
```
Comment (AishwaryShrivastav) [inline src/views/Home.vue:76]: why did we add these id and class?
Please remove if we didn't use it anywhere?

File: src/views/Home.vue [hunk 2]
```diff
@@ -64,9 +64,16 @@
             <ion-button slot="end" fill="outline" size="medium">View all</ion-button>
         </ion-item>
   
-        <div class="scroller-content">
-          <div class="scroller-item" v-for="i = 1 in 10" :key="i">  
-              <ion-card>
+        <div class="scroller-wrapper" ref="ionCard">
```
Comment (AishwaryShrivastav) [inline src/views/Home.vue:67]: Did we use this ref anywhere? If no, then please remove this ref.

File: src/views/Home.vue [hunk 3]
```diff
@@ -126,15 +136,46 @@ export default defineComponent({
...
     }
+  },
+   methods: {
+    async nextItems(event: any) {
+        this.ionCard.scrollLeft += 350;
+        const ionCard = this.ionCard;
+        this.ionCard.style.overflow = 'hidden';
+        const animation = createAnimation()
+        .addElement(this.scrollItem)
+        .easing('linear')
```
Comment (AishwaryShrivastav) [inline src/views/Home.vue:161]: is this necessary?

File: src/views/Home.vue [hunk 4]
```diff
@@ -126,15 +136,46 @@ export default defineComponent({
...
+        .duration(1500)
```
Comment (AishwaryShrivastav) [inline src/views/Home.vue:162]: Are we sure about this duration?

File: src/views/Home.vue (hunk 2 above)
Comment (priyal-hotwax) [reply src/views/Home.vue:67]: Yeah we have used

File: src/views/Home.vue (hunk 3 above)
Comment (priyal-hotwax) [reply src/views/Home.vue:161]: yes.

File: src/views/Home.vue (hunk 4 above)
Comment (priyal-hotwax) [reply src/views/Home.vue:162]: currently I have given this minimum of 1500

File: src/views/Home.vue (hunk 1 above)
Comment (priyal-hotwax) [reply src/views/Home.vue:76]: Will do this fix on next commit.

**Pull request closed without merging**